sdk/textanalytics/azure-ai-textanalytics/tests/test_single.py

```diff
@@ -39,13 +40,13 @@ def test_single_language_detection_bad_credentials(self, resource_group, locatio
         with self.assertRaises(ClientAuthenticationError):
             response = single_detect_language(
                 endpoint=text_analytics_account,
-                credential="xxxxxxxxxxxx",
+                credential=SharedKeyCredential("xxxxxxxxxxxx"),
                 input_text="This is written in English.",
             )
 
     @GlobalTextAnalyticsAccountPreparer()
     def test_single_language_detection_empty_credentials(self, resource_group, location, text_analytics_account, text_analytics_account_key):
-        with self.assertRaises(ClientAuthenticationError):
+        with self.assertRaises(TypeError):
             response = single_detect_language(
                 endpoint=text_analytics_account,
                 credential="",
```

Comment:
Maybe a test for what happens with `SharedKeyCredential("")`

Reply:
added tests